### PR TITLE
Test process/add gsocorg LLR

### PR DESCRIPTION
**Pull request by grasseau**
In the title ..

### CODE REVIEW

**Comment by aflp91** [comment]
Looks OK …

**Comment by JavierCVilla** [comment]
Dear @grasseau, the organization description looks good to me. However, I'm a bit confused with this PR. Are you planning to submit a project proposal later on? If so, do you think you could add it as part of this same PR?

**Comment by grasseau** [comment]
Hi Javier,
Ok It was just a test to remind me the process. Effectively (You or I), we can fuse with the next PR (coming soon - this evening/night).
Thanks
Gilles

**Comment by JavierCVilla** [comment]
@grasseau cool! You can add a project proposal to this PR just pushing your changes to the `master` branch in your fork, they will be automatically added to this PR. 

Feel free to contact me by mail if you need help.

**Comment by grasseau** [comment]
Hi all, (Javier, Antoine)
Have you received the 2 merged PR ?
Thanks !
Gilles

**Comment by grasseau** [comment]
Dear HSF-GSOC Admins,

Comment on the proposal
- very technical proposal but a very good background in DeepLearning, TensorFlow and Keras beacause you have to read and understand the original code - not classical code ... TF/Keras)
- know idea for a good student test (if theoretical ... answers on the net, if technical questions, select not good candidates for us)
- The github HAhRD project is not updated, when would be the dead line for this update (opening GSOC'19 to students) ?
Thanks ... ;-) 
Gilles

File: _gsocproposals/2019/proposal_HAhRD_3D-ObjectDetection.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+already been evaluated in HAhRD project with 2D projections (of our 3D data), thanks
+to a published implementation.
+We propose for GSOC'19 to derivate a 3D version (true 3D input "gray" images), from the original 2D (RGB)
+Mask R-CNN implementation.
+
+
+## Task ideas and expected results
+ * Start with the original implementation on our GPUs platform
+ * Extend to 3D the core of the implementation (ResNet50/ResNet101).
+   Transform RGB values (int) to one gray value (float). Tests/Validation.
```
Comment (JavierCVilla) [inline _gsocproposals/2019/proposal_HAhRD_3D-ObjectDetection.md]: Missing bullet (*)

```suggestion
 * Transform RGB values (int) to one gray value (float). Tests/Validation.
```

**Comment by JavierCVilla** [comment]
> The github HAhRD project is not updated, when would be the dead line for this update (opening GSOC'19 to students) ?

@grasseau You can either update it as part of this PR or later on. 

Interested students should begin discussing project ideas with mentors once Google announces the Organization accepted for this year, that is February 26. However, since some project proposals are already public on our webpage, students might start contacting you.

**Comment by grasseau** [comment]
> 
> 
> @grasseau Looks good to me, I have just commented one minor detail on the list of tasks that you can even edit online.

Ok, thanks

**Pull request merged**